src/app/common/helpers.ts

```diff
@@ -1,7 +1,12 @@
 import dayjs from 'dayjs';
+import advancedFormat from 'dayjs-ext/plugin/advancedFormat';
+import timeZone from 'dayjs-ext/plugin/timeZone';
```

Comment:
@gildub based on the example [here](https://day.js.org/docs/en/plugin/timezone) I think the problem is that you are using a capital Z in `timeZone` and instead you need `timezone`.

Reply:
Thanks. Meanwhile still not working, just showing z character.

Reply:
That's really weird. I'll take a look when I get a chance. I'm also happy to use something other than dayjs in this repo if we want to switch, we don't use it for much. Someone else in UXD recommended https://date-fns.org/